.github/workflows/testBuild.yml

```diff
@@ -4,13 +4,13 @@ on:
   workflow_dispatch:
     inputs:
       # If not specified, only build iOS and Android apps from the main branch of Expensify/App
-      APP_PULL_REQUEST_NUMBER:
-        description: Expensify/App PR number for correct placement of apps. Default to main.
+      APP_PULL_REQUEST_URL:
+        description: The Expensify/App pull request URL (e.g., https://github.com/Expensify/App/pull/12345). Defaults to main.
```

Comment:
I don't know if we need the "Defaults to main." here still, but since we're pulling the PR number from the URL, rather than using the URL explicitly, might be good to leave in there. Please let me know if you have any suggestions for rephrasing this.